doc/README.rst

```diff
@@ -114,12 +114,12 @@ Documentation presentation theme
 ********************************
 
 Sphinx supports easy customization of the generated documentation
-appearance through the use of themes.  Replace the theme files and do
+appearance through the use of themes. Replace the theme files and do
 another ``make htmldocs`` and the output layout and style is changed.
 The ``read-the-docs`` theme is installed as part of the
 ``requirements.txt`` list above, and will be used if it's available, for
-local doc generation.
-
+local doc generation.  :ref:`install_py_requirements` step you took in
```

Comment:
I pushed a `squash!` commit on top with what I was thinking. If that looks OK please feel free to fold it in.

Reply:
Looks good, thanks,  squashed.